modules/customizing-the-appearance/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -1,76 +1,85 @@
 [id='proc-customize-rhdh-sidebar-menuitems_{context}']
 = Customizing the sidebar menu items for your {product-short} instance
 
-The sidebar menu in {product} consists of two main parts:
+The sidebar menu in {product} consists of two main parts that you can configure:
 
-* *Main menu items*: These items are the static menu items that form the core navigation structure of sidebar. These menu items remain consistent and are predefined.
+Dynamic plugin menu items:: Your preferences and your active plugins define dynamically one part of the sidebar menu.
+Main menu items:: The core navigation structure of sidebar is static.
 
-* *Dynamic plugin menu items*: These items are displayed beneath the main menu and can be customized based on the plugins installed. The main menu items section is dynamic and can change based on your preferences and installed plugins.
-
-.Procedure
-
-. Customize the main menu items using the following steps:
-+
---
-.. Open the `app-config-rhdh.yaml` file.
-.. To customize the order and parent-child relationships for the main menu items, use the `dynamicPlugins.frontend.default.main-menu-items.menuItems` field.
-.. For dynamic plugin menu items, use the `dynamicPlugins.frontend.<package_name>.menuItems` field.
-
-.Example `app-config-rhdh.yaml` file
-[source,yaml]
+.Default static menu item configuration
+[source]
 ----
-dynamicPlugins:
-  frontend:
-    <package_name>: # same as `scalprum.name` key in plugin's `package.json`
-      menuItems: # optional, allows you to configure plugin menu items in the main sidebar navigation
-        <menu_item_name>: # unique name in the plugin menu items list <1>
-          icon: home | group | category | extension | school | _<your_icon>_  # <2>
-          title: My Plugin Page # optional, same as `menuItem.text` in `dynamicRoutes` <3>
-          priority: 10 # optional, defines the order of menu items in the sidebar <4>
-          parent: favorites # optional, defines parent-child relationships for nested menu items <5>
+export const DefaultMainMenuItems = {
+  menuItems: {
+    'default.home': {
+      title: 'Home',
+      icon: 'home',
+      to: '/',
+      priority: 100,
+    },
+    'default.my-group': {
+      title: 'My Group',
+      icon: 'group',
+      priority: 90,
+    },
+    'default.catalog': {
+      title: 'Catalog',
+      icon: 'category',
+      to: 'catalog',
+      priority: 80,
+    },
+    'default.apis': {
+      title: 'APIs',
+      icon: 'extension',
+      to: 'api-docs',
+      priority: 70,
+    },
+    'default.learning-path': {
+      title: 'Learning Paths',
+      icon: 'school',
+      to: 'learning-paths',
+      priority: 60,
+    },
+    'default.create': {
+      title: 'Create...',
+      icon: 'add',
+      to: 'create',
+      priority: 50,
+    },
+  },
+};
```

Comment:
Consider translating this into a format that a user could consume. Namely what would the user need to put in the `app-confing.yaml` to configure RHDH with exactly the same settings as the default configuration. With the current information, if we try to apply the steps in the procedure, we are missing a value for the `<plugin_name>`.